LICENSE

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
+                           Version 2.0, January 2004
+                        http://www.apache.org/licenses/
+
+   TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
+
+   1. Definitions.
```

Comment:
@Denes-cilwal Why was the license added?

Reply:
@dipeshdulal  Sir, this was added when we try to use cobra-generator, it adds boilerplate with it, Later it was replaced by the cobra library(manually).